docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
```

Comment:
I'm not exactly sure but this is only for binary tables so it would make sense to highlight ``**binary**`` here, right?